xrayvision/tests/test_clean.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+import pytest
+
+from ..Clean import Hogbom
+from ..Visibility import Visibility
+
+
+class TestClean(object):
+    # Typical map sizes even, odd
+    @pytest.mark.parametrize("", [()])
+    def test_hogbom_intensity_change(self):
+        # Create a visibility bag
+        u = [5, 7, 15, 37, 45, 57, 60]
+        v = [1, 3, 7, 27, 37, 1, 57]
+        uv_in = np.array([u, v]).reshape(2, len(u))
+        vis_in = [5+7j, 1+2j, 10-5j, -6+1j, 7j, -7-7j, 5j]
```

Comment:
You could use real vis abilities using by using the map_dft and fake data like in the tests for that method